### ISSUE TITLE
Installaion problem on Ubuntu 16.04 - Failed building wheel for GDAL

### ISSUE THREAD

**Issue by Omer80** [lucadelu/pyModis#99]
I am trying to install pyModis using pip install. After installation of gdal-bin and python-gdal I still get this error message when trying in install pyModis:
```
...
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
  extensions/gdal_wrap.cpp:3168:22: fatal error: cpl_port.h: No such file or directory
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for GDAL
  Running setup.py clean for GDAL
Failed to build GDAL
...
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
    extensions/gdal_wrap.cpp:3168:22: fatal error: cpl_port.h: No such file or directory
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/home/user/anaconda2/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-QCa82L/GDAL/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-2dKUOZ-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-QCa82L/GDAL/
```
A hint what might be wrong?

**Comment by lucadelu** [comment]
Hi @Omer80,

probably the problem is that you are using Anaconda (right?) and it is like a new environment.

**Comment by lucadelu** [comment]
This should be fixed with GDAL >= 3.2.0 https://pypi.org/project/GDAL/

**Comment by lucadelu** [comment]
Or you can use pygdal
